.travis.yml

```diff
@@ -1,18 +1,55 @@
+sudo: required
+
 language: go
+
 cache:
   directories:
-    - $HOME/.glide/cache
+  - $HOME/.glide/cache
+
 go:
-  - 1.5
-  - 1.6
-  - 1.7rc6
-  - tip
-install: make install_ci
+- 1.5
+- 1.6
+- 1.7rc6
+- tip
+
+services:
+- docker
+
 env:
- # Set higher timeouts for Travis
- - TEST_TIMEOUT_SCALE=20
+  global:
+  - DOCKER_COMPOSE_VERSION=1.8.0
+  - COMMIT=${TRAVIS_COMMIT::8}
+  # Set higher timeouts for Travis
+  - TEST_TIMEOUT_SCALE=20
+  # DOCKER_EMAIL
+  - secure: OS9ccMrvJU1W3LgAltdxSn2iqeC/hS/V+xYZ2yfmGqYifzCWWa2R+GmQAfHaxQ6+ut4ISWJX/EYdtnqoUD3SBYRBOeY1fD47LFX0Q0gFnuxfgf3hS6jG5dpiK9nlp+Vj+fxnWkA2QxQQBzLDkdB1B7mBWBMt+MSuevq1QGxT6coy2XypZoAc2ohvTaznHX4PdISFS2Ddku5Zo7jqL80RYW+K1HeYxnjHYG/K7ZSZbhFQxBM5S9FSvwp8ooGiqQ7kIfK4DsOWiIBXx0fHDNHxJtNJfJeAfF9unqkeNx9QZmGcjMWzbmPqBhz231BRXPd+Z0ZyXZVnlLCzOXN2DsE/JffUTLWTxb6rJ6lrwwZ6oZm/YAM3YHRgjotiC92fjZpXEVoBPGtRMcD+P6iwbedKIeaYOj9XQ5OizKaMbJAwU5HfIUWz5iWW66SZf2NWRPwl0BX8qjL2G3yhwJwCzgIpcZ7zxIbVgtZgMcCG5K5i3/HyuGNGI2vqhXCYoFrqJu+T9t59m74ZyKf76e7bfOUQSGZlUMYVGxRtuCdjMACr4jbHD4lIBSnRGE0uat+OSle/+Ke9plP6W5b8XkALgJreOAgBa5Hnf+0ODeQnCaK0gJk6Ax5O9v4B9NEPae5hvdk1rHvG5ke/MXecu18y7wopMhJ/cpSJxRTfADmuCIdIhy4=
+  # DOCKER_USER
+  - secure: D6KY81H0sxRPvXjE2VeYz1c3vkpEUg/UoV0pRIrVlWD7HOKG6a92ZGUbwz8Ex+PRcTPPlEP1rwE/rJ/0uhrueynkXNBqQsDWT9ECsK7l92b931JY3sEM3eBJCL0whtorCLtXQ2l/g5PJS3AcNGDxP1zP07J2pcsgk2SBcZqnAq0h8U3M7/rLX8JroudG3JxmSvH6HqXkGDdOJXTCeyG9EVdrsUCVb0JY7Q0oMGMzIcZTkJ0Yts7mtjNS/VXDeSYnHxELuRgioLTJThjwBa31StZNTqnnzWh5kR2u/WZvjYf7eaHZfzZJa85CS8yIAAaMc//tu/13qxgAsdiT9TjEhx8hS0J2Wu/JXpCmbKkoHp42sX9aZDWJIR4ZQHaPEIdcTnAbPROhS0XGHqAvpcDAjWuwbtwpnB4Quz67GVpoZNQV4uHiu6fuRqY5kvkzv28K1jGeStUyNDwBtNGCvggB5OZlLYyo+2G7vu//LxV6FXsioqqKePn7xxQBtxgf2LGQEhmeO7iNUyXTy3EoaFrpl7ScQbi9XoCKN28Fi5Da9J5IOn0KQ4niRg126Xgk+vwNytj3qoP4xIZps3WavtTqXvbbGkGCIbIEDNzDFtqrdo2ZAA6YhtlwuB2/2Jbmva7zmma7VDMPapHVqe4tTsQoAl3IEeokV9wRFr/28vGDrKE=
+  # DOCKER_PASS
+  - secure: xzdXvoLEQRRz7x9YLdjyagn83lYgLX9hHlNJy0AnHUfbMpdjbnAzEDBnhorNfqoK/E2Vw4sx13+2M2ZbVQ0qfdjqUne/oBiNxzm+lTUyeZv5lhWeX5ZiJkfmecDmimK/mQBw+eah7ClzPmj3dsL2Oq4E8fd6LVCwO7ZyJUKyaeQ8/cLbJsfOSaBF40T1EFpvX8jim/dz3dtmW5ksHmjPMF3yGKTvSKuz4T2ZCe8CafvLI/Vlei/oNcJuZrIYMXsfSgjAtCW2VNeIgWetezr1KySZXe8rKVre4lyyT48t5OKXqNemxtRXE21pB7td0okpgpYTLfpwP1qP5x/87f3xFEzwBU0iJiTTXaeVifIK0irJwSm8IC63uURg7R0NYjj+183rhQkH37bfKIivj5xBbu/YID8LMTanBNRFkImlHogoSRBRt4ZdcJ7GY2gvN+2+XGWoRmRGwbp5sSHUZtntJoqMDQlu61uU2CX4CuXMy6G6I7um8CX63J0xywqDCbgB2XrQvcyiWRc4PCfnKKBtFKZRBU4aZwf/6r4BckU3NKXE8Sji98OEm2nNpyQmyKR7lfvulPTSWfwfTWUg5SAnzTcBshIsH8rFm8qU8f26WR7YoykgO3PKP/nHVItOkYs/i9roi7az9FX4nEPkgYkgwWV8d/4sJt8Uz+ll1WNIKOE=
+
+before_install:
+- docker version
+- make install_docker_ci
+
+install: make install_ci
+
 script:
- - make test_ci
- - make cover_ci
- - make lint
+- make test_ci
+- make cover_ci
+- make lint
+- make crossdock_ci
 
+after_success:
+- export REPO=tchannelhub/xdock-go
```

Comment:
well, (a) the org `tchannel` is taken, (b) we used `xdock-go` / `xdock-py` image names in Jaeger clients as well because these images actually have nothing to do with tchannel or Jaeger, they are test images, so it seems more appropriate to name them with "crossdock".